docs/source/layouts/_layout.html

```diff
@@ -43,16 +43,23 @@
     └─────────┘
      ↳ http://esri.github.io/calcite-web/patterns/#drawers
      ↳ patterns → _drawers.md
-    -->
-    <div class="drawer drawer-left js-drawer" data-drawer="top-nav">
-      <!-- <nav class="drawer-nav" role="navigation" aria-labelledby="topnavdrawerleft"> -->
+   -->
+   <div class="drawer drawer-left js-drawer" data-drawer="top-nav">
+    <!-- <nav class="drawer-nav" role="navigation" aria-labelledby="topnavdrawerleft"> -->
       <nav class="drawer-nav" role="navigation">
```

Comment:
could just add an inline `aria-label` here. Is `#topnavdrawerleft` missing, is that why you removed the `aria-labelledby`?

Reply:
honestly maybe we should just delete it

Reply:
🚮